handbook/tutorial.md

```diff
@@ -2,28 +2,33 @@
 
 ## Guidelines for Dataset Contributions
 
-We strongly encourage the community of practice to contribute to the expansion of the CAFE Dataverse Collection. Emphasizing open access and collaborative research, the CAFÉ Collection invites contributions from a diverse array of stakeholders, including government agencies, NGOs, community-based organizations, industry partners, and academics. 
+We strongly encourage the community of practice to contribute to the expansion of the CAFE Dataverse Collection. Emphasizing open access and collaborative research, the CAFE Collection invites contributions from a diverse array of stakeholders, including government agencies, NGOs, community-based organizations, industry partners, and academics. 
 
-Parameters for what datasets are appropriate and inappropriate for the CAFE Dataverse are described below:
+Parameters for what datasets are appropriate and inappropriate for the CAFE Collection are described below:
 
 ## General Guidance
 
 - Contributions should be relevant to climate and health research.
 - Contributions should not be identical to data stored in other repositories. The submission of processed derivatives or expansions of data accessible through existing sharing resources (ie: SEDAC, Google Earth Engine) are encouraged.
 - Contributions should be in line with the licensing of raw source data.
 - Data contributors should only post data that they own, have generated, or have been granted permission to reshare in a manipulated version (ie: census data).
-- No restricted access data (ie: data including personal identifying information) should be shared through the CAFE Dataverse. Contributions will be widely accessible to Harvard Dataverse users.
+- No restricted access data (ie: data including personal identifying information) should be shared through the CAFE Collection. Contributions will be widely accessible to Harvard Dataverse users.
 
 ## File Formatting and Size Limitations
 - All file types are supported for upload and download
 - A maximum of 1,000 files are allowed per upload
 - The file upload limit is 300 GB per file
 - Dataverse can ingest data in certain formats as specifically as tabular data, which will allow for exploration and manipulation of the data with external tools. Tabular file ingest is limited to 143.1MB. For more information, see: Dataverse [Tabular Data File Guide](https://guides.dataverse.org/en/6.1/user/dataset-management.html#tabular-data-files) 
```

Comment:
I know this is not part of the new content, but I find this bullet item confusing. If the tabular file limit size correct here?

Reply:
Hey @zpopp I approved and merged the PR, I just left a comment in this paragraph. Do you know what this is referring to?